flink-metric-prometheus/README.md

```diff
@@ -0,0 +1,82 @@
+# Overview
+
+This example shows how to report metrics defined in a FeatureView to a
```

Comment:
This example shows how to define and export application-level metrics for features computed by FeatHub to a Prometheus PushGateway.